challenges/src/main/java/linkedlist/LinkedList.java

```diff
@@ -77,6 +77,75 @@ public String toString(){
 
     }
 
+    // Code Challenge 06 (add to end, before, and after)
+
+    public void addToEnd(int value) {
```

Comment:
try to have your methods match the problem assignment. So instead of addToEnd, use append. It makes it easier for me to find stuff.

Reply:
I will do so going forward, sorry about that.